client/app/hosting/general-informations/GENERAL_INFORMATIONS.controller.js

```diff
@@ -1,18 +1,28 @@
 angular
     .module("App")
     .controller("hostingGeneralInformationsCtrl", class HostingGeneralInformationsCtrl {
-        constructor ($scope, $stateParams, Alerter, hostingSSLCertificate, translator) {
+        constructor ($scope, $stateParams, Alerter, hostingSSLCertificate, HostingRuntimes, translator) {
             this.$scope = $scope;
             this.$stateParams = $stateParams;
 
             this.Alerter = Alerter;
             this.hostingSSLCertificate = hostingSSLCertificate;
             this.translator = translator;
+            this.HostingRuntimes = HostingRuntimes;
         }
 
         $onInit () {
             this.$scope.$on("hosting.ssl.reload", () => this.retrievingSSLCertificate());
 
+            this.defaultRuntime = null;
```

Comment:
You should define `this.loading.defaultRuntime` to `true`, and change it to `false` in `finally`.